lib/nfg/capistrano/tasks/deploy.rake

```diff
@@ -23,14 +23,15 @@ namespace :deploy do
     )
 
     if s3_assets_file.exists?
+      FileUtils.mkdir_p "#{shared_path}/public/assets"
       on release_roles :all do
         if test("[ -f /usr/bin/s3cmd ]")
           execute :s3cmd, "--force get #{s3_bucket}/assets/#{assets_filename} #{shared_path}/public/assets/#{assets_filename}"
         else
           execute :aws, "s3api get-object --profile s3-role --bucket #{s3_bucket.gsub('s3://', '')} --key assets/#{assets_filename} #{shared_path}/public/assets/#{assets_filename}"
         end
         info Airbrussh::Colors.green("Downloaded #{assets_filename} from #{s3_bucket}/assets")
-        execute "tar zxf #{shared_path}/public/assets/#{assets_filename} -C #{shared_path}"
+        execute "tar zxf #{shared_path}/public/assets/#{assets_filename} -C #{release_path}"
```

Comment:
This extracts the compiled assets directly into the release directory, rather than extracting them to the `shared/public/assets` directory and then symlinking them into the release. This ensures that there will never be old assets present in a release.

Reply:
up to you if you want a comment calling this out but makes sense to me